test/loadUpcomingScreenings.test.js

```diff
@@ -0,0 +1,170 @@
+import { describe, expect, jest, test } from "@jest/globals";
+import filterUpcomingScreenings from "../server/filterUpcomingScreenings.js";
+
+describe("loadUpcomingScreenings()", () => {
```

Comment:
The name of this test suite makes me thing it is testing `loadUpcomingScreenings()`, but it's not. It's actually testing `filterUpcomingScreenings()`.